packages/theme/theme.ts

```diff
@@ -44,6 +110,33 @@ export const dark = {
   warning: '#FC540A',
   error: '#E86459',
   underline: 'rgba(229, 229, 229, 0.3)',
+  layout: colors.dark6,
+  btnPrimary: colors.primary5,
+  btnGoogle: colors.red2,
+  btnFont: colors.dark6,
+  btnFacebook: colors.primary9,
+  btnGray: colors.dark1,
+  btnWhite: colors.dark1,
+  text: colors.white1,
+  whiteText: colors.white1,
+  blueText: colors.primary5,
+  darkText: colors.white1,
+  grayText: colors.white1,
+  line: colors.white1,
+  darkLine: colors.gray5,
+  thickLine: colors.primary5,
+  mediumLine: colors.primary8,
+  border: '#EDEDED',
+  grayBorder0: colors.gray1,
+  grayBorder1: colors.white0,
+  grayBorder2: colors.gray4,
+  grayBorder3: colors.gray5,
```

Comment:
Would there be more explicit naming for this? It's hard to tell how to use `0` or `2` at first glance.

Reply:
@hyochan  The intention is that grayBorder 0,1,2 numbers express the degree of darkness.

Reply:
<img width="950" alt="Screen Shot 2020-07-21 at 11 36 16 AM" src="https://user-images.githubusercontent.com/55861805/88006425-8ce21f00-cb46-11ea-9fd5-fc5d9f1079cc.png">

[ref](https://yeun.github.io/open-color/)

Reply:
How about something like borderLight0, 1, 2...

Reply:
That looks better
I will try it